int8 pin
---
float32 ohms #Ohms